src/main/java/frc/robot/commands/ShooterOnBlue.java

```diff
@@ -18,13 +18,13 @@ public ShooterOnBlue(Shooter subsystem) {
     // Called when the command is initially scheduled.
     @Override
     public void initialize() {
+        m_shooter.runTopMotor(blue_top);

```

Comment:
For the motors, it might actually be necessary to set the speed in the execute() method, especially if there are interruptions in communication. No change now, but testing on robot needed.